unit_tests/testHDF5FileType/testHDF5FileType.f90

```diff
@@ -1364,6 +1371,24 @@ SUBROUTINE testCompress()
       !Clear variables
       CALL h5%clear(.TRUE.)
 
+
+      COMPONENT_TEST('Large Array')
+      CALL h5%init('largeData.h5','NEW',.TRUE.)
+      CALL h5%fopen()
+      CALL h5%mkdir('large')
+      CALL TAUSTUB_CHECK_MEMORY()
+      ALLOCATE(largeArray(largeN))
+      DO i=1,largeN
+        largeArray(i)=REAL(i,SDK)*0.0001_SDK
+      ENDDO
+      CALL TAUSTUB_CHECK_MEMORY()
+      CALL t%tic()
+      CALL h5%fwrite('large->array',largeArray)
+      CALL t%toc()
+      WRITE(*,*) "Time to write 128 MB with 4 MB chunks="//t%getTimeHHMMSS()
+      CALL TAUSTUB_CHECK_MEMORY()
+      CALL h5%clear(.TRUE.)
+
     ENDSUBROUTINE testCompress
```

Comment:
Perhaps add an assertion here to make sure that the `large->array` was successfully written to the file?